api/http/templates.go

```diff
@@ -74,229 +74,1311 @@ const bzzList = `{{ define "content" }}
+								<path id="XMLID_2810_" d="M18.9,6.9l0.3-1.7c0.1-0.3-0.2-0.6-0.5-0.6h-3.1l0.6-3.4c0.1-0.3-0.2-0.6-0.5-0.6h-1.7
+									c-0.2,0-0.4,0.2-0.5,0.4L13,4.7H8.9l0.6-3.4C9.6,1,9.3,0.8,9,0.8H7.3c-0.2,0-0.4,0.2-0.5,0.4L6.2,4.7H3c-0.2,0-0.4,0.2-0.5,0.4
+									L2.2,6.8C2.1,7.1,2.4,7.4,2.7,7.4h3.1l-0.9,5.3H1.5c-0.2,0-0.4,0.2-0.5,0.4l-0.3,1.7c-0.1,0.3,0.2,0.6,0.5,0.6h3.1l-0.6,3.4
+									c-0.1,0.3,0.2,0.6,0.5,0.6h1.7c0.2,0,0.4-0.2,0.5-0.4L7,15.3h4.1l-0.6,3.4c-0.1,0.3,0.2,0.6,0.5,0.6h1.7c0.2,0,0.4-0.2,0.5-0.4
+									l0.6-3.6H17c0.2,0,0.4-0.2,0.5-0.4l0.3-1.7c0.1-0.3-0.2-0.6-0.5-0.6h-3.1l0.9-5.3h3.3C18.7,7.4,18.9,7.2,18.9,6.9z M11.6,12.6H7.5
+									l0.9-5.3h4.1L11.6,12.6z"/>
+								</svg>
+							</button>
+						</div>
+					</div>
+					<div id="uploadFeedbackBarsWrapper">
+						<div id="uploadFeedbackBars">
+							<div class="uploadFeedbackBar uploadFeedbackColor1" id="uploadReceivedBar"></div>
+							<div class="uploadFeedbackBar uploadFeedbackColorx hidden" id="uploadSeenBar"></div>							
+							<div class="uploadFeedbackBar uploadFeedbackColor2" id="uploadSplitBar"></div>
+							<div class="uploadFeedbackBar uploadFeedbackColor3" id="uploadStoredBar"></div>
+							<div class="uploadFeedbackBar uploadFeedbackColor4" id="uploadSentBar"></div>							
+						</div>
+						<div class="incrementLine incrementLine1"></div>
+						<div class="incrementLine incrementLine2"></div>
+						<div class="incrementLine incrementLine3"></div>
+					</div>
+					<div class="uploadFeedbackCounts">
+						<div class="uploadFeedbackCount uploadFeedbackCountColor1">Uploaded <span class="uploadFeedbackCountNumbers" id="uploadReceivedCount"></span></div>
+						<div class="uploadFeedbackCount uploadFeedbackCountColor2">Stored <span class="uploadFeedbackCountNumbers" id="uploadStoredCount"></span></div>
+						<div class="uploadFeedbackCount uploadFeedbackCountColor3 hidden">Seen <span class="uploadFeedbackCountNumbers" id="uploadSeenCount"></span></div>						
+						<div class="uploadFeedbackCount uploadFeedbackCountColor3">Split <span class="uploadFeedbackCountNumbers" id="uploadSplitCount"></span></div>
+						<div class="uploadFeedbackCount uploadFeedbackCountColor5">Sent <span class="uploadFeedbackCountNumbers" id="uploadSentCount"></span></div>
+					</div>
+					<div class="clearBoth"></div>
+				</div>
+			</div>
+		</div>
+		<input type="text" id="uploadLinkInput" class="invisible"/>
+		<input type="text" id="uploadHashInput" class="invisible"/>{{ end }}`
 
-const baseTemplate = `<html>
+const baseTemplate = `
+<!DOCTYPE html>
+<html>
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0"/>
-  <meta http-equiv="X-UA-Compatible" ww="chrome=1"/>
-  <link rel="icon" type="image/x-icon" href="favicon.ico"/>
-  <style>
-  {{ template "css" . }}
-  </style>
+	<title>Swarm Gateway</title>
```

Comment:
I do not think that this page is meant only for gateways, but for any swarm node. Maybe to remove the work Gateway? Also from the svg logo?

Reply:
i think this would be worth some discussion - for me 'gateway' is a contraction of 'http gateway' so this is semantically accurate, but i'm open to being convinced otherwise! 😉